tests/data/AnsysSound_FRF_2024R2_20241206.txt

```diff
@@ -0,0 +1,30 @@
+AnsysSound_FRF 1
```

Comment:
@ansaminard we should we use a realistic FRF here (this file looks like a spectrum)

Reply:
@anssmolla For testing purpose, it does not matter, this one does the trick.
Now if you insist to only have realistic data everywhere, then I feel there are many other test data that don't meet this requirement. And I would also argue that, on the contrary it is interesting to have, in a testing context, rather extreme data cases (after all this how we found the bug on the filter coefficients in comparison with SAS)

Reply:
@anssmolla fyi, these are the files we're using in the C++ test suite

Reply:
Ok I get the point, and eventually it does not look so "unrealistic". Let's keep it as is. And keep it mind fro the future to use realistic data, even in unit tests.